main.nf

```diff
@@ -357,9 +414,46 @@ process dupradar {
                          expressionHist(DupMat=dm)
     title("Distribution of RPK values per gene")
     dev.off()
+    file.create("dup.done")
+    """
+
+    }else{
+
+    """
+    #!/usr/bin/env Rscript
+    if (!("dupRadar" %in% installed.packages()[,"Package"])){
+        .libPaths( c( "${params.rlocation}", .libPaths() ) )
+        source("https://bioconductor.org/biocLite.R")
+        biocLite("dupRadar")
+    }
+    library("dupRadar")
+          
+    # Duplicate stats
+    stranded <- 2
+    paired <- FALSE
```

Comment:
Again, if this is the only line that changes, would be nicer to have this in a variable instead of duplicating the whole script..